userspace/engine/falco_common.h

```diff
@@ -0,0 +1,69 @@
+#pragma once
+
+#include <string>
+#include <exception>
+
+extern "C" {
+#include "lua.h"
+#include "lualib.h"
+#include "lauxlib.h"
+}
+
+#include <sinsp.h>
+
+//
+// Most falco_* classes can throw exceptions. Unless directly related
+// to low-level failures like inability to open file, etc, they will
+// be of this type.
+//
+
+struct falco_exception : std::exception
+{
+	falco_exception()
+	{
+	}
+
+	~falco_exception() throw()
```

Comment:
whenever class is meant to be inherited from, destructor should be virtual. I'd remove that destructor exception specification, too.